samples/net/wifi/test_certs/install_certs.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script installs Wi-Fi certificates to a device using the `device_credentials_installer` tool.
+#
+# shellcheck disable=SC2086,SC2154
+# Usage:
+#   ./install_certs.sh <path_to_certs> [port] [mode]
+#
+# Arguments:
+#   <path_to_certs>  Path to the directory containing the certificate files.
+#   [port]           (Optional) Serial port to use for communication with the device. Default is
+#                    /dev/ttyACM1.
+#   [mode]           (Optional) Mode of operation: AP or STA. Default is STA.
+#
+# Dependencies:
+#   - `device_credentials_installer` tool must be installed. You can install it using:
+#     pip3 install nrfcloud-utils
```

Comment:
Does this mean that the instructions for installing the certs are only available for Nordic devices?

Reply:
No, that tools works with non-Nordic devices too, verified with STM32. BTW, this script should be part of the other PR will remove from here.